core/admin/mailu/api/v1/__init__.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint
+from flask_restx import Api, fields
+
+
+VERSION = 1.0
+api_token = None
+
+blueprint = Blueprint(f'api_v{int(VERSION)}', __name__)
+
+authorization = {
+    'Bearer': {
+        'type': 'apiKey',
+        'in': 'header',
+        'name': 'Authorization'
+    }
+}
+
+api = Api(
+    blueprint, version=f'{VERSION:.1f}',
+    title='Mailu API', default_label='Mailu',
+    validate=True,
+    authorizations=authorization,
+    security='Bearer',
+    doc='/swaggerui/'
+)
```

Comment:
Is there a way to redirect /api/v1/ to /api/v1/swaggerui/ if there is no parameter ?

Reply:
I am just worried that if you browse to /api/ you get redirected to /api/v1/ that displays nothing useful. IMHO the documentation or a placeholder listing versions would be more useful.

Reply:
/api goes to the login page. 
/api/ will redirect to /api/v1./ /api/v1 will show the documentation.